src/FormModelInterface.php

```diff
@@ -19,6 +19,15 @@ interface FormModelInterface extends DataSetInterface, FormMetadataInterface
      */
     public function attributes(): array;
 
+    /**
+     * Check for attribute exists in model
+     *
+     * @param string $attribute
+     *
+     * @return bool
+     */
+    public function hasAttribute(string $attribute): bool;
```

Comment:
It affects BC, can we postpone for a next major release?

Reply:
Yes. But without that it have psalm test error

Reply:
Could you please show it?

Reply:
This package has no release so it's fine to add it to the interface.

Reply:
> Could you please show it?

Of course. That was happend after remove this method from [validator](https://github.com/yiisoft/validator/) 9 days ago

![Снимок экрана от 2022-03-05 11-09-15](https://user-images.githubusercontent.com/90403480/156874746-16a59e34-1637-41bb-bcd5-92e3a0192c01.png)

Reply:
Yes, it's a side effect of https://github.com/yiisoft/validator/pull/177. I already added it in #183, but adding in this or a separate PR is also fine. No worries for BC break since validator, form and even request model do not have a stable release yet.

Reply:
Created a separate PR to not wait until others are merged. https://github.com/yiisoft/form/pull/185